### PR TITLE
Move Logsearchapi binary to Operator Container Image

### PR DESCRIPTION
**Pull request by dvaldivia**
Signed-off-by: Daniel Valdivia <18384552+dvaldivia@users.noreply.github.com>
Fixes https://github.com/miniohq/engineering/issues/342

### CODE REVIEW

File: pkg/apis/minio.min.io/v2/constants.go [hunk 1]
```diff
@@ -147,7 +147,7 @@ const PrometheusHLSvcNameSuffix = "-prometheus-hl-svc"
 // Log related constants
 
 // DefaultLogSearchAPIImage specifies the latest logsearchapi container image
-const DefaultLogSearchAPIImage = "minio/logsearchapi:v4.4.4"
+const DefaultLogSearchAPIImage = "minio/operator:v4.4.4"
```
Comment (harshavardhana) [inline pkg/apis/minio.min.io/v2/constants.go:150]: shouldn't this be `v4.4.5` ? @dvaldivia

**Review by harshavardhana — CHANGES_REQUESTED** [review]

File: pkg/apis/minio.min.io/v2/constants.go (hunk 1 above)
Comment (dvaldivia) [reply pkg/apis/minio.min.io/v2/constants.go:150]: it will become `v4.4.5` on that release

File: Dockerfile [hunk 2]
```diff
@@ -18,5 +18,6 @@ RUN \
     microdnf install curl ca-certificates shadow-utils --nodocs
 
 COPY minio-operator /minio-operator
+COPY logsearchapi/logsearchapi /logsearchapi
```
Comment (harshavardhana) [inline Dockerfile:21]:
```suggestion
COPY logsearchapi /logsearchapi
```

**Review by harshavardhana — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 2 above)
Comment (dvaldivia) [reply Dockerfile]: done

**Review by harshavardhana — APPROVED** [review]

**Pull request merged**